app/controllers/expenditures_controller.rb

```diff
@@ -0,0 +1,72 @@
+class ExpendituresController < ApplicationController
+  before_action :authenticate_user!
+  before_action :set_expenditure, only: %i[show edit update destroy]
+
+  # GET /expenditures or /expenditures.json
+  def index
```

Comment:
- [ ] From the requirements of this project, for a given category, the list of transactions is presented ordered by the most recent. let's try to fix this issue. Your list of transactions is not presented in this order. When I add a transaction, it is positioned at the bottom instead of the top.